docs/eventing/sugar/README.md

```diff
@@ -0,0 +1,358 @@
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-abc123
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+```
+
+If filters are added to the Service,
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+  annotations:
+    trigger.eventing.knative.dev/filter.attributes: |
+      [{"type":"gumdrops", "sticky":"yes"}]
+```
+
+A Trigger with filters will replace the original:
+
+```yaml
+kind: Trigger
+...
+spec:
+  ...
+  filter:
+    attributes:
+      type: gumdrops
+      sticky: "yes"
+```
+
+### Multiple Triggers Example
+
+More than one Trigger could be configured in the filter annotation, when the
+following Knative Service is created:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+  annotations:
+    trigger.eventing.knative.dev/filter.attributes: |
+      [{"type":"gumdrops"}, {"subject":"lollipops"}]
+spec:
+  template:
+    spec:
+      containers:
+        - image: <some image>
+```
+
+Two Triggers will be created like:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-abc123
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: gumdrops
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-efg456
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  filter:
+    attributes:
+      subject: lollipops
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+```
+
+### Broker Named Trigger Example
+
+Using the `autotrigger.eventing.knative.dev/broker` annotation, the Addressable
+resource can control which Broker name is used in the spec of the managed
+Trigger (spec.broker), when the following Knative Service is created:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+  annotations:
+    autotrigger.eventing.knative.dev/broker: gloppy
+    autotrigger.eventing.knative.dev/filter.attributes: |
+      [{"type":"gumdrops"}]
+spec:
+  template:
+    spec:
+      containers:
+        - image: <some image>
+```
+
+A Trigger using the Broker name of "gloppy" will be created like:
```

Comment:
@n3wscott this example and use case is great to have a knative service or any addresable be able to create the trigger with a broker named, but I this scenario how can I get the broker `gloppy` created? I would think would be something to add that will add the annotation to the triggers created?
```
annotation:
  autotrigger.eventing.knative.dev/broker: gloppy
```

I can't create the broker `gloppy` because namespace annotation doesn't allow me to name the broker is always named `default`

Reply:
Ok, second time this has been requested...

Reply:
How do you feel about https://github.com/knative/docs/pull/3040/files#diff-a5afdc5012c7e77bf6fb45370fa61c4547612113c1aac0b1a8bf37500ae94f5fR369

Reply:
Nice that makes a full example. 👍 

My guess is the the service is the owner/parent  ref of the trigger created and the trigger is the owner/parent ref of the broker. 
So deleting the service starts the cascading effect ?